.NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexFormat.cs

```diff
@@ -64,47 +65,43 @@ public static string Format(TimexProperty timex)
 
         private static string FormatDuration(TimexProperty timex)
         {
-            var isTimeDurationAlreadyExist = false;
-            var timexBuilder = new StringBuilder(Constants.GeneralPeriodPrefix);
+            var timexList = new List<string> { };
             if (timex.Years != null)
             {
-                timexBuilder.Append($"{timex.Years}Y");
+                timexList.Add($"{Constants.GeneralPeriodPrefix}{timex.Years}{Constants.TimexYear}");
```

Comment:
All generation of timexes should come from methods that receive the parameters needed to create it.
Only that method should know which string to use.